lib/model/card/action/end_game_action.dart

```diff
@@ -14,6 +14,7 @@ class EndGameAction extends CardAction {
   @override
   void performActionImpl(GameState state) {
     state.result = this.result;
+    state.endGame();
```

Comment:
Would probs be tempted to refactor to:

```suggestion
    state.endGame(result);
```